keyboards/handwired/tritium_numpad/keymaps/max/keymap.c

```diff
@@ -0,0 +1,100 @@
+#include QMK_KEYBOARD_H
+
+#ifdef RGBLIGHT_ENABLE
+#include "rgblight.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.